### PR TITLE
Feature: Get metadata for a Python package

### PR DESCRIPTION
**Pull request by pacospace**
- Introduced the get python package metadata for a Python Package

Depends-On: https://github.com/thoth-station/storages/pull/1110

Specification for metadata are taken from: https://packaging.python.org/specifications/core-metadata/

Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -1767,3 +1811,85 @@ components:
...
+        - maintainer_email
+        - metadata_version
+        - name
+        - platform
+        - requires_dist
+        - summary
+        - version
+      properties:
+        author:
+          type: string
```
Comment (fridex) [inline openapi/openapi.yaml]: Please allow nullable values for all properties.

**Comment by pacospace** [comment]
> One minor comment, otherwise we can release new thoth-storages and this can go in.
> 
> BTW we will need to design API endpoints in the upcoming PRs.

Perfect :) Do you want to create a card for that?

**Comment by fridex** [comment]
> > One minor comment, otherwise we can release new thoth-storages and this can go in.
> > BTW we will need to design API endpoints in the upcoming PRs.
> 
> Perfect :) Do you want to create a card for that?

Card added to the new column.

**Comment by fridex** [comment]
@pacospace BTW for next PRs - we should handle also 404 (if the package is not found in the database). Now we will be reporting 500 (internal server errors), which is not nice.

**Comment by pacospace** [comment]
> @pacospace BTW for next PRs - we should handle also 404 (if the package is not found in the database). Now we will be reporting 500 (internal server errors), which is not nice.

I changed it in the PR for schema :)

**Pull request merged**